src/ai/backend/manager/models/image.py

```diff
@@ -301,7 +301,7 @@ async def rescan_images(
     # TODO: delete images removed from registry?
 
 
-type Resources = dict[SlotName, dict[str, Decimal]]
+type Resources = dict[SlotName, dict[str, Any]]
```

Comment:
Is there a reason you changed this? @jopemachine

Reply:
It's very strange that it's updated because of the test code, is that correct?

Reply:
It seems that I added an incorrect type in the recent PR I submitted.